package.json

```diff
@@ -34,8 +34,7 @@
     "extend": "~3.0.2",
     "forever-agent": "~0.6.1",
     "form-data": "~2.3.2",
-    "har-validator": "~5.1.3",
```

Comment:
we might want to keep this for now

Reply:
It's deprecated, and it wasn't actually tested, which is why this didn't show up in a simple test.